staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -233,11 +233,11 @@ func TestEvents(t *testing.T) {
 		store.Update(makeTestPod("pod", uint64(i)))
 	}
 
-	// Test with full cache - there should be elements from 5 to 9.
+	// cache grows to capacity 10, all elements 3 to 9 all stored.
 	{
-		_, err := store.GetAllEventsSince(3)
-		if err == nil {
-			t.Errorf("expected error too old")
+		events, err := store.GetAllEventsSince(3)
+		if err != nil || len(events) != 6 {
+			t.Errorf("expected event length %d but get %d, error %v", 6, len(events), err)
```

Comment:
I would prefer not to change existing tests.
So instead, we should add ability to overwrite max in tests and do that here.

Reply:
Done by setting lowerBoundCapacity and upperBoundCapacity as 5